src/include/utils/palloc.h

```diff
@@ -105,6 +119,26 @@ extern volatile OOMTimeType* segmentOOMTime;
 extern volatile OOMTimeType oomTrackerStartTime;
 extern volatile OOMTimeType alreadyReportedOOMTime;
 
+#ifdef EXTRA_DYNAMIC_MEMORY_DEBUG
+#define MAX_TOP_ALLOC_CHUNK_STATS 10
```

Comment:
I think this parameter we might expose as GUC. It's for your choice

Reply:
I think this parameter we could stay as define, because to enable feature we have to uncomment define EXTRA_DYNAMIC_MEMORY_DEBUG and have to reconfigure gpdb. Before reconfigure we can to change this parameter.